### PR TITLE
Implement Active Storage

### LINKED ISSUE #1: Active Storage
## REQUIREMENTS / DESCRIPTION

> Implementing active storage function to be able to upload media
      - Define media config
      -  Related Active Storage with the necessary table
      - Use the attachment's [download](https://edgeapi.rubyonrails.org/classes/ActiveStorage/Blob.html#method-i-download) 
          method to read a blob’s binary data into memory
     - Attaching Files to Records

### PR DESCRIPTION
**Pull request by angel23v**
## Quick Info

Configure all the necessary requirements requested by Active Storage
Attaching all the files to Records in the necessary models

Migrations? 👎
Yes

## What does this change?
Now you can use media files in the project

## Why are you changing that?
Because using media files is necessary for the users

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -23,6 +23,10 @@
   # Disable serving static files from `public/`, relying on NGINX/Apache to do so instead.
   # config.public_file_server.enabled = false
 
+  # Store files on Amazon S3.
+  config.active_storage.service = :amazon
```
Comment (juperameza) [inline config/environments/production.rb:27]: You're gonna configure Amazon in production??

**Review by juperameza — CHANGES_REQUESTED** [review]

File: config/environments/production.rb (hunk 1 above)
Comment (angel23v) [reply config/environments/production.rb:27]: According to [Active Storage](https://edgeguides.rubyonrails.org/active_storage_overview.html) it is configured there. 
Or what do u mean?

**Review by Antonio3veces3 — APPROVED** [review]
Well done!

File: config/environments/production.rb (hunk 1 above)
Comment (juperameza) [reply config/environments/production.rb:27]: Take into consideration that when you deploy you need to ctually configure the amazon s3 bucket

**Review by juperameza — APPROVED** [review]

**Pull request merged**